### PR TITLE
Update Set-CsTeamsNetworkRoamingPolicy.md

### PR DESCRIPTION
**Pull request by vini-gupta**
Update users to enable the network roaming policy for users who are not enterprise voice enabled in TeamsMeetingPolicy.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

File: skype/skype-ps/skype/Set-CsTeamsNetworkRoamingPolicy.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ The TeamsNetworkRoamingPolicy cmdlets enable administrators to provide specific
 
 More on the impact of bit rate setting on bandwidth can be found [here](https://docs.microsoft.com/microsoftteams/prepare-network).
 
+To enable the network roaming policy for users who are not enterprise voice enabled, you must also enable the AllowNetworkConfigurationSettingsLookup setting in TeamsMeetingPolicy. This setting is off by default. See Set-TeamsMeetingPolicy for more info on how to enable AllowNetworkConfigurationSettingsLookup for non-enterprise voice enabled users.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsNetworkRoamingPolicy.md:33]:
```suggestion
To enable the network roaming policy for users who are not Enterprise Voice enabled, you must also enable the AllowNetworkConfigurationSettingsLookup setting in TeamsMeetingPolicy. This setting is off by default. See Set-TeamsMeetingPolicy for more information on how to enable AllowNetworkConfigurationSettingsLookup for users who are not Enterprise Voice enabled. 
```

Comment (JohanFreelancer9) [reply skype/skype-ps/skype/Set-CsTeamsNetworkRoamingPolicy.md:33]: Kindly insert a link to Set-TeamsMeetingPolicy if you want to refer to it.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @vini-gupta, kindly commit the suggested changes. Many thanks.

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Comment by yogkumgit** [comment]
@tseward please review

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**